lib/constants/enums.ts

```diff
@@ -1,7 +1,6 @@
 /** An enumeration of payment channel network clients that support token swaps. */
 export enum SwapClientType {
   Lnd,
-  Raiden,
```

Comment:
I wonder if this breaks existing xud environments since in the database it's 0, 1, 2, and now 0, 1?